drivers/shared/executor/executor_universal_linux.go

```diff
@@ -226,7 +234,11 @@ func (e *UniversalExecutor) configureCG2(cgroup string, command *ExecCommand) {
 	// write memory cgroup files
 	memHard, memSoft := e.computeMemory(command)
 	ed := cgroupslib.OpenPath(cgroup)
-	_ = ed.Write("memory.max", strconv.FormatInt(memHard, 10))
+	if memHard == memoryNoLimit {
+		_ = ed.Write("memory.max", "max")
```

Comment:
just noting here that "max" is a magic word indicating no limit, ref:

https://git.kernel.org/pub/scm/linux/kernel/git/tj/cgroup.git/tree/Documentation/admin-guide/cgroup-v2.rst#n667

> Limits are in the range [0, max] and defaults to "max", which is noop.